adafruit_rfm9x.py

```diff
@@ -721,6 +588,18 @@ def enable_crc(self, val):
                 self._read_u8(_RH_RF95_REG_1E_MODEM_CONFIG2) & 0xFB,
             )
 
+    def tx_done(self):
+        """Transmit status"""
+        return (self._read_u8(_RH_RF95_REG_12_IRQ_FLAGS) & 0x8) >> 3
+
+    def rx_done(self):
+        """Receive status"""
+        return (self._read_u8(_RH_RF95_REG_12_IRQ_FLAGS) & 0x40) >> 6
+
+    def crc_error(self):
+        """crc status"""
+        return (self._read_u8(_RH_RF95_REG_12_IRQ_FLAGS) & 0x20) >> 5
```

Comment:
How much memory does this save? Seems like it'd very close to the cost of three instances of `_RegisterBits`.

Reply:
The main saving is that register bits creates the getter and setter -- these are all read only so they are never set. still -- it's not huge savings.

Reply:
The getter and setter functions should be shared though because it's three instances of the same class.

Reply:
I don’t understand. There is no setter.

Reply:
Are you saying that with registerbits the unused setter dies not cost anything?

Reply:
Yes. The function objects in the data descriptor class (registerbits) should be shared amongst all instances if my mental model is correct.